tracing/tracers/lightstep/lightstep.go

```diff
@@ -3,6 +3,7 @@ package lightstep
 import (
 	"errors"
 	"fmt"
+	"github.com/zalando/skipper/logging"
```

Comment:
I think you should use plain github.com/sirupsen/logrus, instead.

Reply:
Sure, I was not sure which one as it is used in a mixed way in different parts of the code. Will send an update.